README.md

```diff
@@ -45,6 +45,19 @@ client = AblyRest('api:key')
 channel = client.channels.get('channel_name')
 ```
 
+You can define the logging level for the whole library, and override for an
```

Comment:
I'd split it in:

- here's how you set the level
- if you like you can even set a custom handler